documentation/index.qmd

```diff
@@ -0,0 +1,79 @@
+<a href="https://tiledb.com"><img src="https://github.com/TileDB-Inc/TileDB/raw/dev/doc/source/_static/tiledb-logo_color_no_margin_@4x.png" alt="TileDB logo" width="400"></a>
+
+[![TileDB-BioImaging CI](https://github.com/TileDB-Inc/TileDB-BioImaging/actions/workflows/ci.yml/badge.svg)](https://github.com/TileDB-Inc/TileDB-BioImaging/actions/workflows/ci.yml)
+![Coverage Badge](https://img.shields.io/endpoint?url=https://gist.githubusercontent.com/ktsitsi/32d48185733a4e7375e80e3e35fab452/raw/gist_bioimg.json)
+
+# TileDB-BioImaging
+
+Python package for:
+
+- converting images stored in popular Biomedical Imaging formats to groups of TileDB arrays (& vice versa)
+- exposing an expressive and efficient API (powered by TileDB) for querying such data.
+
+**Note**: this project is in pretty early stage and under heavy development.
+Breaking changes to the underlying data format and the API are to be expected.
+
+## Features
+
+### Ingestion to TileDB Groups of Arrays
+    - OME-Zarr
+    - OME-Tiff
+    - Open-Slide
+
+### Export from TileDB-Bioimaging Arrays to:
+    - OME-Zarr
+    - OME-Tiff
+
+### Visualization Options
+
+- [TileDB Cloud](https://cloud.tiledb.com) includes a built-in, pyramidal multi-resolution viewer (log in to TileDB Cloud to see an example image preview [here](https://cloud.tiledb.com/biomedical-imaging/TileDB-Inc/dbb7dfcc-28b3-40e5-916f-6509a666d950/preview))
+- Napari: https://github.com/TileDB-Inc/napari-tiledb-bioimg
+
+## Quick Installation
+
+- From PyPI:
+
+      # If you use zsh replace [full] with \[full\]
+      pip install tiledb-bioimg[full]
+
+- From source:
+
+      git clone https://github.com/TileDB-Inc/TileDB-BioImaging.git
+      cd TileDB-BioImaging
+
+      # If you use zsh replace .[full] with .\[full\]
+      pip install -e .[full]
+
+**Note:** For `tiff` support (`[tiff]` or `[full]` extra dependencies) on Python 3.7, you
```

Comment:
We've dropped support for Python 3.7 everywhere else -- do we even need to mention it here?